spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Lilly', photo: 'https://example.com/image.jpg', bio: 'Teacher from Poland.') }
+
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'post count should be equal to or greater than 0' do
+    subject.posts_count = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'post count should be a number' do
+    subject.posts_count = 'hello'
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Good job with test coverage, but since it is a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md)for this project, "`Add unit specs for all of your models' methods and validations`", please ensure all custom methods in the models have a corresponding unit test, methods like: `increment_posts_count!`, `recent_posts`. Please kindly fix that.